kernel-cert-script

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh
+
+op=$(echo "$0" | sed -e "s/.*-//")
+
+while true ; do
+    case $1 in
+	--kmp)
+	    kmp="$2"
+	    shift 2
```

Comment:
Why split the shift in 2?

Only options with parameter are supported which renders shift 2 when the option is processed.

There is no need to shift when an option is not processed so there is no point to have shift outside of the case.

Why are kmp or certs supposed to be initialized to empty?

Or should all variables that come as options be initialized to empty in case they leak from the environment?

Reply:
> Only options with parameter are supported which renders shift 2 when the option is processed.

Perhaps we'll add parameter-less options in the future? 

> There is no need to shift when an option is not processed so there is no point to have shift outside of the case.

Well, I'd appreciate if you could do me the favor to do it this way. Shifting the options outside the `case` statement is a common idiom that makes the code more fool-proof and easier to read, for me at least. But I won't reject the PR if you don't.
 
> Why are kmp or certs supposed to be initialized to empty?

To avoid them being initialized from random variables from the environment. It's good practice in shell scripting.

> Or should all variables that come as options be initialized to empty in case they leak from the environment?

That's the idea, yes. Either that, or use well defined environment variables with expressive names for their initialization.

Reply:
The arguments accepted are pretty much final.

If at later point a kernel that passes more arguments exists it does not remove the kernels that don't from the repositories.

Reply:
As I said, it won't be a reason to reject this PR even if I still have other style preferences. I can deal with it :-)